fem/quadinterpolator.hpp

```diff
@@ -53,38 +54,43 @@ class QuadratureInterpolator
    static const int MAX_ND3D = 1000;
    static const int MAX_VDIM3D = 3;
 
-public:
    enum EvalFlags
    {
       VALUES       = 1 << 0,  ///< Evaluate the values at quadrature points
       DERIVATIVES  = 1 << 1,  ///< Evaluate the derivatives at quadrature points
       /** @brief Assuming the derivative at quadrature points form a matrix,
           this flag can be used to compute and store their determinants. This
           flag can only be used in Mult(). */
-      DETERMINANTS = 1 << 2
+      DETERMINANTS = 1 << 2,
+      PHYSICAL_DERIVATIVES = 1 << 3 ///< Evaluate the physical derivatives
    };
 
    QuadratureInterpolator(const FiniteElementSpace &fes,
-                          const IntegrationRule &ir);
+                          const IntegrationRule &ir,
+                          const bool use_tensor_products = false);
```

Comment:
I prefer to keep the constructor parameters to a minimum. Any extra parameters should be controlled by their own methods -- this makes the code more readable.

Reply:
`done`